lib/openshift/console_notification_console_openshift_io.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class ConsoleNotificationConsoleOpenshiftIo < ClusterResource
+    RESOURCE = "consolenotification.console.openshift.io"
```

Comment:
I can't find any such object in stage cluster but I think it must be `consolenotifications.console.openshift.io` with `s` at the end. This is how I see other APIs to be. If your cluster has such an object, then you can check for sure. What I see is that the `s` works:
```
🐚 oc --config=/tmp/kubeconfig get consolenotifications.console.openshift.io
No resources found.
```

That's why I suggested to put the `s` at the end.

To be sure, can you post here `oc get -o yaml` from such an object?

Reply:
@xiaocwan  Please check new comments